packages/network/lib/index.ts

```diff
@@ -4,6 +4,7 @@ import * as connect from './connect'
 import * as cors from './cors'
 import * as httpUtils from './http-utils'
 import * as uri from './uri'
+import * as pki from './pki'
```

Comment:
i know we already named this from `networkPki` to `pki`, but in line with the earlier comment, this code module should probably be called `clientCertificates`, not `pki`